store/migration/10001__init_schema.sql

```diff
@@ -1425,3 +1425,33 @@ WHERE
     rowid = old.rowid;
 
 END;
+
+-- query table stores the saved queries for the user
+CREATE TABLE query (
```

Comment:
Also we should do this schema change file in a separate PR

1. Schema change should normally be done in a separate PR.

2. This PR causes more confusing because the code change serves a different purpose (for query history) than than the schema change (for user saved query)